legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -94,7 +102,7 @@ def get_in_dissolution_batch_processing(business_id: int):
             one_or_none()
 
     @staticmethod
-    def _get_businesses_eligible_query(exclude_in_dissolution=True):
+    def _get_businesses_eligible_query(exclude_in_dissolution=True, exclude_future_effective_filing=False):
         """Return SQLAlchemy clause for fetching businesses eligible for involuntary dissolution.
 
         Args:
```

Comment:
Maybe consider updating the Args part of the docString for the newly added `exclude_future_effective_filing` argument?